internal/cmd/generate/commands/gentests/command.go

```diff
@@ -85,7 +86,7 @@ type Command struct {
 //
 func (cmd *Command) Execute() error {
 	if len(apiRegistry) < 1 {
-		return fmt.Errorf("API registry in 'api_registry.gen.go' is empty: Did you run go generate?")
+		return errors.New("API registry in 'api_registry.gen.go' is empty: Did you run go generate?")
```

Comment:
As I said in the other comment, the `internal/cmd` code is so hairy that I would just skip it for cosmetic changes at this stage...